web/src/components/KebabMenu.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { ReactElement, ReactNode, useState } from 'react'
+import styled from 'styled-components'
+
+import { UiDirectionType } from 'translations'
+
+import iconClose from '../assets/IconClose.svg'
+import iconKebabMenu from '../assets/IconKebabMenu.svg'
+import dimensions from '../constants/dimensions'
+import '../styles/KebabMenu.css'
+import { Portal } from './Portal'
+
+type KebabMenuProps = {
+  items: Array<ReactNode>
+  direction: UiDirectionType
+}
+
+const Toggle = styled.div`
+  display: flex;
+  padding: 0 12px;
+  z-index: 32;
+`
+
+const List = styled.div<{ direction: UiDirectionType; checked: boolean }>`
+  font-family: ${props => props.theme.fonts.web.decorativeFont};
+  position: absolute;
+  top: 0;
+  width: 80vw;
+  height: 100vh;
+  background-color: ${props => props.theme.colors.backgroundColor};
+  box-shadow: -3px 3px 3px 0 rgba(0, 0, 0, 0.13);
+  -webkit-font-smoothing: antialiased;
+  /* to stop flickering of text in safari 
+   */
```

Comment:
Is the comment in the right place or does it belong to line 31?

Reply:
moved above